From a6dd458a12ad8db161271e2271644803d4f29a96 Mon Sep 17 00:00:00 2001
From: Douglas Fenstermacher <douglas.fenstermacher@gmail.com>
Date: Wed, 20 Jan 2021 10:47:56 -0500
Subject: [PATCH] fixes Github Actions failures (#4548)

Github Actions failing due to new release of pyyaml==5.4, this change should resolve that issue
---
 networkx/readwrite/nx_yaml.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/networkx/readwrite/nx_yaml.py b/networkx/readwrite/nx_yaml.py
index 80bd5dfd0c..98b4b1d9cb 100644
--- a/networkx/readwrite/nx_yaml.py
+++ b/networkx/readwrite/nx_yaml.py
@@ -102,5 +102,5 @@ def read_yaml(path):
     except ImportError as e:
         raise ImportError("read_yaml() requires PyYAML: http://pyyaml.org/") from e
 
-    G = yaml.load(path, Loader=yaml.FullLoader)
+    G = yaml.load(path, Loader=yaml.Loader)
     return G
